test/parallel/test-eslint-crypto-check.js

```diff
@@ -13,20 +13,32 @@ new RuleTester().run('crypto-check', rule, {
     'foo',
     'crypto',
     `
-      if (!common.hasCrypto) {
-        common.skip();
-      }
-      require('crypto');
+    if (!common.hasCrypto) {
+      common.skip();
```

Comment:
Does this (and other occurences below) pass the test? I think the `'missing crypto'` message is missing.

Reply:
Added the string `'missing crypto'` in tests. But there's still something wrong. Any help?

Reply:
Done now.